Cargo.toml

```diff
@@ -14,7 +14,7 @@ path = "bin/rvps/src/main.rs"
 required-features = [ "rvps-server", "rvps-proxy", "tokio/rt-multi-thread" ]
 
 [features]
-default = [ "rvps-server", "in-toto" ]
+default = [ "rvps-server", "in-toto", "rvps-proxy", "tokio/rt-multi-thread" ]
```

Comment:
By default we do not need `rvps-proxy` and `tokio/rt-multi-thread` feature to be enabled.
My naive idea is to replace
```bash
cargo build $(release)
```
in line 20 of `Makefile`
```bash
cargo build --bin grpc-as --features rvps-server,rvps-proxy,tokio/rt-multi-thread
```

How do you like?

Reply:
If there multiple binaries are expected to be built, like it was `grpc-as grpc-as-ctl`, then cargo will repeat the command to build each of them with the enabled features.

Reply:
I thought about this

```bash
cargo build --bin grpc-as --features rvps-server,rvps-proxy,tokio/rt-multi-thread
```

Just think it might not flexible, but I am fine with the approach as well.

Reply:
`cargo build` will only build the lib `attestation-service`. 
- `tokio/rt-multi-thread` is only used when an outer caller of the lib wants to use `tokio::main` or some other runtimes.
- `rvps-server` and `rvps-proxy` usually will not be used at the same time, excepte for `grpc-bin`.
For the two reasons, a specific build command would be better, I think.

> If there multiple binaries are expected to be built, like it was `grpc-as grpc-as-ctl`, then cargo will repeat the command to build each of them with the enabled features.

Yes. Some binaries might not depend all these features of AS, which will help to reduce the resulted binary size.

Reply:
It seems that it is better for us to regard the compilation of each binary as a make option? Something like:

```
grpc-as:
    cargo build --bin grpc-as --features rvps-server,rvps-proxy,tokio/rt-multi-thread
```

So that we can build `grpc-as` with:

```
make grpc-as
```

Reply:
+1, updated, thanks for the suggestion.